docs/content/.vuepress/styles/index.scss

```diff
@@ -84,7 +84,6 @@ html[data-theme='light'] {
 
       .vp-hero-infos {
         font-family: 'Sharp Sans';
-        max-width: 25rem;
```

Comment:
We want to get rid of the docs homepage entirely, in the meantime this makes it look like:
![image](https://github.com/dxos/dxos/assets/1310143/2ed89ea1-eda7-431a-8d50-aa2fdcb53e4b)

Reply:
(We can point the docs link on dxos.org to `docs.dxos.org/composer`, so the only way to get here will be clicking the DXOS icon in the top left of the docs.)

Reply:
the three links should probably be:
- composer docs
- sdk docs
- api docs

Can we make the link in the top-left corner point to https://dxos.org?

Reply:
Yes, those links. Sounds good!

Reply:
Done.